sdk/servicebus/service-bus/src/log.ts

```diff
@@ -82,3 +82,8 @@ export const semaphore = debugModule("azure:service-bus:semaphore");
  * log statements for sessionManager
  */
 export const sessionManager = debugModule("azure:service-bus:sessionManager");
+/**
+ * @internal
+ * log statements for Atom XML management API over HTTP
+ */
+export const httpAtomXml = debugModule("http:service-bus:atom-xml");
```

Comment:
I think this should be changed, at least to start with `azure` in the namespace. This would make it easy to enable all logs for this package by specifying `DEBUG=azure*` or `DEBUG=azure:service-bus:*`.

Alternatively, we could name this `azure:service-bus:info` (and create separate `error`, `warn` namespaces as needed) to better conform with #4926. I don't feel strongly though if we want to put that off until preview 2 so we can move all the log namespaces at once.

Reply:
I agree with the suggestion to start with `azure:`, so something like `azure:service-bus:atom` or `azure:service-bus:http` should suffice.

I would hold off on the info vs warn vs error until Track 2

Reply:
> One main feedback is that `options` should be strongly typed in all the public methods.
Addressed this in current wave of changes per offline discussion with Chris